.gitignore

```diff
@@ -5,6 +5,7 @@ node_modules
 .task/
 *.swp
 .DS_Store
+.vscode
```

Comment:
Please add editor specific files to your local [global git ignore](https://stackoverflow.com/questions/7335420/global-git-ignore) list, so we can prevent this from becoming a pit.